<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>safe_guard</class>
 <widget class="QMainWindow" name="safe_guard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(246, 245, 244);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(192, 191, 188);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>查看当前规则</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(192, 191, 188);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>更新规则</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(192, 191, 188);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>清空规则</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(192, 191, 188);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>使用说明</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(192, 191, 188);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>查看日志</string>
    </property>
   </widget>
   <widget class="QListView" name="view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>741</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border: 1px solid black</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示拦截</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>修改日志路径</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
